src/lib/edje/edje_calc.c

```diff
@@ -3262,7 +3271,6 @@ _edje_vector_load_lottie(Edje *ed, Edje_Real_Part *ep, const char *key)
         efl_gfx_frame_controller_frame_set(ep->object, (int)(frame_count * desc->vg.frame));
      }
 }
-#endif
```

Comment:
Why changing where the `endif` of  `#ifdef BUILD_VG_LOADER_JSON`?

Reply:
Because it makes not much sense to have vector functions if vg_loader is disabled, I think, but looking at the code I think this was a misunderstanding.

Reply:
For me, it seems like a misunderstanding too, I mean, It was there for a reason, and if it was wrong or not shouldn't be decided at this PR.

Reply:
I'll address that in the next commit.

Reply:
Done.